src/runtime-rs/crates/shim/src/args.rs

```diff
@@ -52,21 +51,6 @@ impl Args {
         validate::verify_id(&self.id).context("verify container id")?;
         validate::verify_id(&self.namespace).context("verify namespace")?;
 
-        // Ensure `address` is a valid path.
-        let path = PathBuf::from(self.address.clone())
-            .canonicalize()
-            .context(Error::InvalidPath(self.address.clone()))?;
-        let md = path
-            .metadata()
-            .context(Error::FileGetMetadata(format!("{:?}", path)))?;
-        if !md.file_type().is_socket() {
```

Comment:
This is the last user of `os::unix::fs::FileTypeExt` as detected by CI :
```
error: unused import: `os::unix::fs::FileTypeExt`
 --> crates/shim/src/args.rs:7:11
  |
7 | use std::{os::unix::fs::FileTypeExt, path::PathBuf};
  |           ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `-D unused-imports` implied by `-D warnings`
```

Reply:
I've never understood how an idea such as treating warnings as errors managed not to die immediately... _sigh_ I'll fix it.